vision_datasets/common/dataset/dataset.py

```diff
@@ -25,8 +25,9 @@ def __init__(self, transform=None):
         ImageFile.LOAD_TRUNCATED_IMAGES = True
 
     @property
-    def labels(self):
-        """Returns a list of labels."""
+    @abstractclassmethod
+    def categories(self):
```

Comment:
This interface change has some impacts. Can we keep `labels` property, too?

Reply:
there are actually more impacts coming from the introduction of image label manifest, that would require updates on client side. Should we still keep the labels?